doc/code/qml_spin.rst

```diff
@@ -7,40 +7,20 @@ Overview
 This module contains functions and classes for creating and manipulating Hamiltonians for
 spin models.
 
-Hamiltonian functions
-^^^^^^^^^^^^^^^^^^^^^
+Conversion functions
+^^^^^^^^^^^^^^^^^^^^
 
-.. currentmodule:: pennylane.spin
+.. currentmodule:: pennylane.io
 
 .. autosummary::
     :toctree: api
 
-    ~emery
-    ~fermi_hubbard
-    ~haldane
-    ~heisenberg
-    ~kitaev
-    ~transverse_ising
-
-
-Hamiltonian custom functions
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-.. currentmodule:: pennylane.spin
-
-.. autosummary::
-    :toctree: api
-
-    ~spin_hamiltonian
-
-
-Lattice classes and functions
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-.. currentmodule:: pennylane.spin
-
-.. autosummary::
-    :toctree: api
-
-    ~Lattice
-    ~generate_lattice
```

Comment:
Merge conflict?

Reply:
Somehow I accidentally changed this...? Ill revert it.